orchestrator/settings.py

```diff
@@ -48,7 +48,7 @@ class AppSettings(BaseSettings):
     EXECUTOR: str = ExecutorType.THREADPOOL
     WORKFLOWS_SWAGGER_HOST: str = "localhost"
     WORKFLOWS_GUI_URI: str = "http://localhost:3000"
-    DATABASE_URI: PostgresDsn = "postgresql://nwa:nwa@localhost/orchestrator-core"  # type: ignore
```

Comment:
Any chance of making this change backwards compatible that will save a lot of people some pain.

Reply:
The whole reason for this MR is to switch to `psycopg` v3 so we can't assume the `psycopg2` module is available at all. We could convert `postgresql://` connection strings to `postgresql+psycopg://` and perhaps `postgresql+psycopg2://` as well? Perhaps issuing a warning that the conversion happened and the `DATABASE_URI` should be fixed.

Reply:
Yes the conversion and the warning would be great, that gives some time for people to migrate

Reply:
thanks @RichieB2B! I ran some tests internally, it seems to work 👍🏼